apps/swirl-docs/src/components/Tokens/TokenItem.tsx

```diff
@@ -7,23 +15,38 @@ export type TokenItemProps = {
 
 const TokenItem = ({ token }: TokenItemProps) => {
   return (
-    <tr className="grid gap-2 grid-cols-1 md:grid-cols-5 items-start p-4 border-b-1">
+    <tr
+      className={classNames(
+        `grid gap-2 grid-cols-1 md:grid-cols-typography-token-list items-center py-4 border-b-1`,
+        {
+          "md:grid-cols-color-token-list":
+            isColorIndex(token.type) || isBorderToken(token.type),
+          "md:grid-cols-typography-token-list": isTypographyToken(token.type),
+          "md:grid-cols-spacing-token-list": isSpacingToken(token.type),
+          "md:grid-cols-z-index-token-list": isZindexToken(token.type),
+        }
+      )}
+    >
+      <td>
+        <TokenPreview token={token} />
+      </td>
       <td className="col-span-2 flex flex-col mb-2 md:mb-0">
         <div className="inline-flex mb-2 md:mb-0">
-          <TokenPreview token={token} />
-          <div className="flex flex-col items-start">
-            <code className="w-full max-w-[176px] bg-gray-100 rounded-md p-1 text-sm font-font-family-code">
-              {token.name}
+          <div className="flex flex-col items-start max-w-[100%]">
+            <code className="w-full whitespace-pre overflow-hidden text-ellipsis bg-gray-100 rounded-md p-1 text-sm font-font-family-code">
+              <dfn className="not-italic" title={token.name}>
```

Comment:
**issue:** Not really familiar with the dfn element, but I think it should point to its definition via [aria-details](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details) (the `<code>` tag in line 45 and the `td` in line 48, I guess).